### PR TITLE
Fixed primitives.

### PR DESCRIPTION
**Pull request by akallabeth**
Reenabled ASM optimized decoder paths.

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test

**Comment by akallabeth** [comment]
@freerdp-bot test

File: libfreerdp/primitives/prim_YUV.c [hunk 1]
```diff
@@ -549,8 +549,8 @@ static INLINE BYTE RGB2U(INT32 R, INT32 G, INT32 B)
 
 static INLINE BYTE RGB2V(INT32 R, INT32 G, INT32 B)
 {
-	const INT32 v = (128L * (R) - 116L * (G) -  12L * (B));
-	const INT32 v8 = (v >> 8L) + 128L;
+	const INT32 v = (128L / 4 * (R) - 116L / 4 * (G) -  12L / 4 * (B));
+	const INT32 v8 = (v >> 6L) + 128L;
 	return CLIP(v8);
```
Comment (nfedera) [inline libfreerdp/primitives/prim_YUV.c]: Please elaborate: Aren't you losing precision here?

Comment (akallabeth) [inline libfreerdp/primitives/prim_YUV.c]: @nfedera No loss of precision here, just trimmed the whole equation the lowest common denominator

Comment (akallabeth) [inline libfreerdp/primitives/prim_YUV.c]: @nfedera So after checking the maximum, this one equation could be done with signed 16bit values...

**Comment by akallabeth** [comment]
@nfedera anything preventing a merge?

File: winpr/libwinpr/sysinfo/cpufeatures/repo.prop [hunk 2]
```diff
@@ -0,0 +1,55 @@
+platform/bionic ed9e6a41c92c9552be84ecc126e29b4604eee246
```
Comment (bmiklautz) [inline winpr/libwinpr/sysinfo/cpufeatures/repo.prop]: Is the repo.prop still usable? I mean is it possible to update the code still? If not we should remove it.

**Comment by bmiklautz** [comment]
I'd opt in to remove Android.mk and add a Readme instead where the Android detection code is from and how it can possibly be updated.

**Pull request merged**